platform/commonUI/dialog/res/templates/overlay.html

```diff
@@ -19,7 +19,7 @@
  this source code distribution or the Licensing information page available
  at runtime from the About dialog for additional information.
 -->
-<div class="abs overlay">
+<div class="abs overlay" ng-class="{'delayEntry100ms' : !!ngModel.delay}">
```

Comment:
I'm a simple man, and double negative logic hurts my brain. Angular will handle the `undefined` case gracefully, so I'd be inclined to remove the `!!`